docs/external-secrets.md

```diff
@@ -0,0 +1,53 @@
+---
+title: External secrets
+description: Use an external secrets vault with n8n.
+contentType: howto
+---
+
+# External secrets
+
+!!! info "Feature availability"
+	* External secrets are available on Enterprise Self-hosted and Cloud plans.
+	* n8n supports Infisical and HashiCorp Vault. 
+	* n8n doesn't support [HashiCorp Vault Secrets](https://developer.hashicorp.com/hcp/docs/vault-secrets){:target=_blank .external-link}.
+
+You can use an external secrets store to manage credentials for n8n.
+
+## Connect n8n to your secrets store
+
+1. In n8n, go to **Settings** > **External Secrets**.
+1. Select your store provider.
```

Comment:
What about: 

Select **Set Up** for the store provider you want to enable.

to make it clearer that they have to click on the "Set Up" button?